### DISCUSSION TITLE
Does the test-attestaion-service command only work on local machines?

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/social-connect#233] [Help (Q&A)]
Moving a question from @rootwarp on [Discord](https://discord.com/channels/600834479145353243/654086275900375041/988346609811193876) over here:

> Hey. I have a question to test switching CELO_PROVIDERS.
> 1. Add  two RPCs on CELO_PROVIDERS. Local and https://forno.celo.org/.
> 2. And I stopped local node temporary.
> 3. Tried to send SMS by celocli identity:test-attestation-service command.
> 
> So far, I expected to receive test SMS but CLI command returned error with Error: Invalid JSON RPC response: "".
> Does test-attestaion-service command works only on local machine? 

![image](https://user-images.githubusercontent.com/46296830/174780250-0a2ee4d6-a099-446e-b405-544b50a9d555.png)

**Comment by alecps** [comment]
When you say local node what do you mean exactly? If you stopped your attestation signer node, then it would make sense that you would not receive a response. I don't believe the /test-attestations endpoint even calls CELO_PROVIDERS, so you shouldn't really notice a difference based on that command if all you're doing is changing CELO_PROVIDERS. I will confirm though just to be sure

**Reply by arthurgousset** [reply]
cc @rootwarp

**Reply by rootwarp** [reply]
> 

Yup. I stopped attestation signer node which is running with attestation service.

As you said, */test-attestations* endpoint does not use *CELO_PROVIDERS* so I can't test *CELO_PROVIDERS* is working by using */test-attestations* endpoint.
Am I understand well?

**Reply by eelanagaraj** [reply]
When your service is starting up, it should check your signing configuration & CELO_PROVIDERS and throw an error then. If you have your service up and running, and get a response when you hit the `/status` endpoint of your local node), your providers should be configured properly.

As far as the `/test_attestations` issue you're seeing, can you provide more information? Which `SMS_PROVIDERS` were you testing this for? Can you provide the attestation service logs and/or send over the non-sensitive (i.e. no API keys!!) parts of your service configuration? Thanks!
